releases/v19.2.0-beta.20191014.md

```diff
@@ -0,0 +1,94 @@
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v19.2.0-beta.20191014
+~~~
+
+### SQL language changes
+
+- The new `SHOW RANGE ... FOR ROW` statement identifies the range containing a specific row of a table or secondary index. This is particularly useful for validating the location of a specific piece of data when using [partitions](../v19.2/topology-geo-partitioned-replicas.html) to control data location for improved performance or GDPR compliance. [#40954][#40954]
+- The [`SHOW PARTITIONS`](../v19.2/show-partitions.html) statement now displays inherited as well as explicitly set zone configurations for partitions. Also, it is now possible to retrieve full zone configurations via the `full_config_sql` and `full_config_yaml` columns in the `crdb_internal.zones` table, and the new `zone_id` and `subzone_id` columns in the `crdb_internal.partitions` table point to partitions' corresponding zone configurations in `crdb_internal.zones`. [#40493][#40493]
+- Renamed the `config_yaml`, `config_sql`, and `config_proto` columns in the `crdb_internal.zones` table to `raw_config_yaml`, `raw_config_sql`, and `raw_config_proto`. [#40493][#40493]
+
+### Command-line changes
+
+- When `vmodule` is set to `exec_logs=2` to [log queries executed by a node](query-behavior-troubleshooting.html#per-node-execution-logs), CockroachDB-generated statements are no longer logged alongside client-generated statements. To log CockroachDB-generated statements as well, you must now use `exec_logs=3`. [#41149][#41149]
+- The [`cockroach demo`](../v19.2/cockroach-demo.html) command with `--nodes` set to `4` or higher runs an in-memory, multi-node cluster across multiple default regions. To simulate latencies between these default regions, you can now add the `--global` flag. Note that this flag is experimental and, as such, is hidden from the `cockroach demo --help` output and may change in future releases. [#40811][#40811]
+
+### Bug fixes
+
+- Fixed a bug that prevented changes to the `kv.bulk_io_write.concurrent_addsstable_requests` [cluster setting](../v19.2/cluster-settings.html) from taking effect. [#41195][#41195]
+- Fixed a memory leak that affected secondary logging (SQL audit logs, statement execution, and RocksDB logging). [#41231][#41231]
+- Fixed a bug where zone configuration changes on tables with existing index zone configurations would not take effect unless the `num_replicas` field was also set. [#41261][#41261]
+- Fixed a bug where zone configuration changes on secondary indexes wouldn't perform setting verification, such as constraints with lease preferences or setting only one of `min_range_size` and `max_range size`. [#41216][#41216]
+- Fixed a rare data corruption bug in RocksDB caused by newer Linux kernels' handling of `i_generation` on certain filesystems. [#41346][#41346]
+- Fixed a bug that could cause transactions to not see their own previous writes. [#41307][#41307]
```

Comment:
This bug sounds very serious and it may be useful to include some details about when the situation would previously arise, to avoid alarming unconcerned users unnecessarily.